### PR TITLE
fix: aligned dev with release/20210514

### PR DESCRIPTION
**Pull request by IvanJobs**
Since some PRs were not merged back to dev branch, add the missing changes in this PR.

### CODE REVIEW

**Review by 1yefuwang1 — APPROVED** [review]

File: packages/sdk/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/teamsfx",
-  "version": "0.1.2-rc.4",
+  "version": "0.1.3",
```
Comment (tecton) [inline packages/sdk/package.json:3]: Does SDK have any changes that need to bump version?

File: packages/cli/package.json [hunk 2]
```diff
@@ -89,11 +88,11 @@
     "@azure/arm-resources": "^4.0.0",
     "@azure/arm-sql": "^7.0.2",
     "@azure/arm-subscriptions": "^3.0.0",
-    "@azure/core-http": "^1.2.0",
-    "@azure/identity": "^1.3.0",
-    "@azure/ms-rest-nodeauth": "^3.0.6",
-    "@microsoft/teamsfx-api": "^0.1.0-rc.4",
-    "@microsoft/teamsfx-core": "^0.1.0-rc.4",
+    "@azure/core-http": "^1.2.2",
+    "@azure/identity": "^1.2.3",
```
Comment (chagong) [inline packages/cli/package.json:92]: I am ok with upgrade to newer version for dependencies. But a little bit concern about downgrade the dependencies. dev probably upgrades to newer version for a reason.

File: packages/sdk/package.json (hunk 1 above)
Comment (IvanJobs) [reply packages/sdk/package.json:3]: I don't have the knowledge of this. May have a offline discussion first.

File: packages/cli/package.json [hunk 3]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@microsoft/teamsfx-cli",
-  "version": "0.1.0-rc.4",
-  "author": "Microsoft Corporation",
+  "version": "0.1.1",
```
Comment (chagong) [inline packages/cli/package.json:3]: What's the policy of package version on dev branch? I think merge back might need version update.

Comment (LongOddCode) [reply packages/cli/package.json:3]: +1

Comment (IvanJobs) [reply packages/cli/package.json:3]: Thanks for the info and let's have a discussion offline since I don't have enough knowledge about this.

**Review by tecton — APPROVED** [review]

**Review by swatDong — APPROVED** [review]
LGTM

**Review by eriolchan — APPROVED** [review]

**Review by chagong — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: templates/msgext/js/default/README.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ A Messaging Extension allows users to interact with your web service while compo
 
 ## Create an application
 
-- From Visual Studio Code, open command palette and select `Teams: Create New Project`. Or from the CLI, (after `npm install -g @microsoft/teamsfx-cli`) run command `teamsfx new`.
+- From Visual Studio Code, open command palette and select `Teams - Start a new project`. Or from the CLI, (after `npm install -g @microsoft/teamsfx-cli`) run command `teamsfx new`.
```
Comment (MuyangAmigo) [inline templates/msgext/js/default/README.md:13]: Hi @IvanJobs, the latest command palette is "Teams: Create New Project", it should not be changed.

**Pull request merged**